libs/analytics/WordPressAnalytics/src/main/java/org/wordpress/android/analytics/AnalyticsTracker.java

```diff
@@ -269,6 +270,13 @@ public static void track(Stat stat, Map<String, ?> properties) {
         }
     }
 
+    public static void track(Stat stat, String errorContext, String errorType, String errorDescription) {
```

Comment:
It would be helpful to have a comment header explaining what each of these parameters is.